BUILDING.md

```diff
@@ -1,244 +1,102 @@
 ```
-
-Note that the task ("ami") must come **after** the arguments to `cargo make` that are specified with `-e`.
-
-Your new AMI ID will be printed after it's registered.
-
-If you built your image for a different architecture or variant, just use the same arguments here:
-
-```shell
-cargo make -e PUBLISH_REGIONS=your-region-here -e BUILDSYS_VARIANT=my-variant-here ami
+[vendor.my-vendor]
+registry = "####.dkr.ecr.us-west-2.amazonaws.com"
+[[kit]]
+name = "bottlerocket-core-kit" # Name of your ECR repo
+version = "2.x.y" # your version tag you want to test
+vendor = "my-vendor"
 ```
-
-(There's a lot more detail on building and managing AMIs in the [PUBLISHING](PUBLISHING.md) guide.)
-
-## Use your image
-
-See any of the setup guides tailored to the various execution environments for information on running Bottlerocket images:
-
-* [Setup guide for Kubernetes](QUICKSTART-EKS.md)
-* [Setup guide for Amazon ECS](QUICKSTART-ECS.md)
-* [Setup guide for VMware](QUICKSTART-VMWARE.md)
-* [Setup guide for QEMU/KVM](QUICKSTART-LOCAL.md)
-
-## Publish your image
-
-See the [PUBLISHING](PUBLISHING.md) guide for information on deploying Bottlerocket images and repositories.
-
-## Building out-of-tree kernel modules
-
-To further extend Bottlerocket, you may want to build extra kernel modules.
-The specifics of building an out-of-tree module will vary by project, but the first step is to download the "kmod kit" that contains the kernel headers and toolchain you'll need to use.
-
-### Downloading the kmod kit
-
-kmod kits are included in the official Bottlerocket repos starting with Bottlerocket v1.0.6.
-Let's say you want to download the kit for building x86_64 modules for v1.11.0 and variant aws-k8s-1.24.
-
-First, you need tuftool:
-```shell
-cargo install tuftool
+Any time you change the vendor and/or version of the kit above you need to run twoliter update to update the Twoliter.lock
 ```
-
-Next, you need the Bottlerocket root role, which is used by tuftool to verify the kmod kit.
-This will download and verify the root role itself:
-```shell
-curl -O "https://cache.bottlerocket.aws/root.json"
-sha512sum -c <<<"a3c58bc73999264f6f28f3ed9bfcb325a5be943a782852c7d53e803881968e0a4698bd54c2f125493f4669610a9da83a1787eb58a8303b2ee488fa2a3f7d802f  root.json"
+twoliter update
 ```
-
-Next, set your desired parameters, and download the kmod kit:
-```shell
-ARCH=x86_64
-VERSION=v1.11.0
-VARIANT=aws-k8s-1.24
-OUTDIR="${VARIANT}-${VERSION}"
-
-tuftool download "${OUTDIR}" --target-name ${VARIANT}-${ARCH}-kmod-kit-${VERSION}.tar.xz \
-   --root ./root.json \
-   --metadata-url "https://updates.bottlerocket.aws/2020-07-07/${VARIANT}/${ARCH}/" \
-   --targets-url "https://updates.bottlerocket.aws/targets/"
+### Build a Variant
+Now you can build a variant, ami etc using the standard well known cargo make commands such as,
```

Comment:
I would leave this part out since it's not related to building or publishing a core kit.